contracts/Registry.sol

```diff
@@ -679,4 +680,12 @@ contract Registry is IRegistry {
         orgNames = typeReposByTag[tag].orgNames;
         repositoryNames = typeReposByTag[tag].itemNames;
     }
+
+    // ERC165: https://eips.ethereum.org/EIPS/eip-165
+
```

Comment:
NIT: newline between comment and function comment refers to

Reply:
addressed